src/apps/main/antivirus/windows-defender/parse-virus-names.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Parses the output from MpCmdRun.exe to extract actual virus names
+ * @param stdout Standard output from MpCmdRun.exe
+ * @param stderr Standard error from MpCmdRun.exe
+ * @param isInfected Whether the file is infected
+ * @returns Array of virus names found
+ */
+export function parseVirusNames(stdout: string, stderr: string, isInfected: boolean): string[] {
```

Comment:
For consistency, we usually pass parameters as an object. It improves readability at the call site, avoids issues with parameter order, and makes the function easier to extend in the future.

You could do something like
```suggestion
interface Props {stdout: string, stderr: string, isInfected: boolean}

export function parseVirusNames({ stdout, stderr, isInfected }: Props) {
```